demo/src/docs/ColorPicker.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Formik, Form, Field } from "formik";
+
+import Paper from "elevate-ui/Paper";
+import ColorPicker from "elevate-ui/ColorPicker";
+import Button from "elevate-ui/Button";
+
+const Inputs = () => (
```

Comment:
This should match your component name, so I would expect it to be `ColorPickers`.